From 01f759ea8786c39867228fd3a4fbf1a84de4c393 Mon Sep 17 00:00:00 2001
From: Udayan Banerji <udayan.banerji@intel.com>
Date: Tue, 26 Jun 2012 16:19:55 -0700
Subject: Dalvik: Fix errors due to turning on JIT_TUNING

BZ: 47554

This change correctly loads arguments for call to dvmBumpNoChain, enabling
WITH_JIT_TUNING flag to be turned on.

Category: device-enablement
Domain: AOSP-Dalvik-Runtime
Origin: internal
Upstream-Candidate: no, needs rework

Change-Id: I78c7aa3b90417701515ea2209d92ef86613e2d5a
Orig-MCG-Change-Id: Ife9c54a7524bfcc56d2754308b9947db832a1b30
Signed-off-by: Udayan Banerji <udayan.banerji@intel.com>
Signed-off-by: Qiming Shi <qiming.shi@intel.com>
Signed-off-by: Serguei Katkov <serguei.i.katkov@intel.com>
---
 vm/Android.mk |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/vm/Android.mk b/vm/Android.mk
index 66951a5..f0c3d69 100644
--- a/vm/Android.mk
+++ b/vm/Android.mk
@@ -166,7 +166,7 @@ ifeq ($(WITH_HOST_DALVIK),true)
     # TODO: split out the asflags.
     LOCAL_ASFLAGS := $(LOCAL_CFLAGS)
     # To enable assert in host mode, uncomment the following line
-    # LOCAL_CFLAGS += -UNDEBUG -DDEBUG=1 -DLOG_NDEBUG=1 -DWITH_DALVIK_ASSERT
+    # LOCAL_CFLAGS += -UNDEBUG -DDEBUG=1 -DLOG_NDEBUG=1 -DWITH_DALVIK_ASSERT -DWITH_JIT_TUNING
     LOCAL_CFLAGS += -Wa,--noexecstack
     LOCAL_MODULE_TAGS := optional
     LOCAL_MODULE := libdvm
-- 
1.7.4.1

